resources/views/emails/internal/coverageChangedNotification.blade.php

```diff
@@ -21,12 +21,10 @@
 @endif
 
 <p>
-	Hierdoor is de vakdekking voor het kamp <strong>{{ ($statusAfter) ? 'verbeterd' : 'verslechterd' }}</strong>, klik <a href="{{ url('events', [$event->id, 'check', 'all']) }}">hier</a> om die te bekijken.
+	Hierdoor is de <a href="{{ url('events', [$event->id, 'check', 'all']) }}" target="_blank">vakdekking voor het kamp</a> <strong>{{ ($statusAfter) ? 'verbeterd' : 'verslechterd' }}</strong>.
 </p>
 
 <p>
 	Met vriendelijke groet,<br/>
 	AAS 2.0
```

Comment:
Maybe we can centralize this "footer" in the config or something?

Reply:
Heb de 'subject prefix' wel configurabel gemaakt maar dit vind ik wat overdreven, gaat niet veranderen denk ik